pkg/sqlite/load.go

```diff
@@ -757,29 +790,29 @@ func (s *sqlLoader) AddBundleSemver(graph *registry.Package, bundle *registry.Bu
 	return nil
 }
 
-func (s *sqlLoader) AddBundlePackageChannels(manifest registry.PackageManifest, bundle *registry.Bundle) error {
-	tx, err := s.db.Begin()
-	if err != nil {
-		return err
-	}
-	defer func() {
-		tx.Rollback()
-	}()
+// func (s *sqlLoader) AddBundlePackageChannels(manifest registry.PackageManifest, bundle *registry.Bundle) error {
+// 	tx, err := s.db.Begin()
+// 	if err != nil {
+// 		return err
+// 	}
+// 	defer func() {
+// 		tx.Rollback()
+// 	}()
 
-	if err := s.addOperatorBundle(tx, bundle); err != nil {
-		return err
-	}
+// 	if err := s.addOperatorBundle(tx, bundle); err != nil {
+// 		return err
+// 	}
 
-	if err := s.rmPackage(tx, manifest.PackageName); err != nil {
-		return err
-	}
+// 	if err := s.rmPackage(tx, manifest.PackageName); err != nil {
+// 		return err
+// 	}
 
-	if err := s.addPackageChannels(tx, manifest); err != nil {
-		return err
-	}
+// 	if err := s.addPackageChannels(tx, manifest); err != nil {
+// 		return err
+// 	}
 
-	return tx.Commit()
-}
+// 	return tx.Commit()
+// }
```

Comment:
Note: commented this out because it doesn't seem to be used anywhere

Reply:
If it's unused, let's delete it entirely.